annotation/alembic/versions/7cc1ed83c309_compare_agreement_scores.py

```diff
@@ -0,0 +1,38 @@
+"""compare agreement scores
+
+Revision ID: 7cc1ed83c309
+Revises: 9083dea3783e
+Create Date: 2022-12-29 17:23:44.079258
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7cc1ed83c309"
+down_revision = "9083dea3783e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "agreement_metrics",
+        sa.Column("task_from", sa.INTEGER(), nullable=False),
+        sa.Column("task_to", sa.INTEGER(), nullable=False),
+        sa.Column("agreement_metric", sa.FLOAT(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["task_from"], ["tasks.id"], ondelete="cascade"
+        ),
+        sa.ForeignKeyConstraint(["task_to"], ["tasks.id"], ondelete="cascade"),
+        sa.PrimaryKeyConstraint("task_from", "task_to"),
```

Comment:
May you adjust migration to the corresponding ORM model in `models.py`? More specifically `agreement_metric`is nullable here, but not in the model. I believe we don't need a record with null metric.
And may you specify `unique=False` here as well?

Reply:
Actually `agreement_metric` was nullable in models and also was nullable here.  But i agree we don't need this column nullable so it's fixed.
`unique=False` it is not an alembic syntax. if you want to specify unique column into alembic migration you should use `sqlalchemy.UniqueConstraint`, since we don't use `UniqueConstraint` here those columns aren't unique.

Reply:
Don't we use SQLAlchemmy `Column` here? They should accept `unique=False`.

Reply:
It has `unique=False` by default
__
https://github.com/epam/badgerdoc/blob/main/assets/alembic/versions/afa33cc83d57_new_fields.py#L34